engine/src/main/java/io/zeebe/engine/processing/streamprocessor/ProcessingContext.java

```diff
@@ -156,13 +165,13 @@ public TransactionContext getTransactionContext() {
   }
 
   @Override
-  public CommandResponseWriter getCommandResponseWriter() {
-    return commandResponseWriter;
+  public BooleanSupplier getAbortCondition() {
+    return abortCondition;
   }
 
   @Override
-  public BooleanSupplier getAbortCondition() {
-    return abortCondition;
+  public EventApplier getEventApplier() {
```

Comment:
Do we still want to offer this method? Or do we just want to have `getWriters()`? Personally, I slightly prefer the latter option

Reply:
The EventApplier must be available for the ReprocessingStatemachine